react-rendering-lists/my-app/src/PokemonList.js

```diff
@@ -0,0 +1,12 @@
+export default function pokemonList({pokedex}) {
```

Comment:
Start this with uppercase. You use an uppercase name when you import it, but for consistency, name the function with uppercase, also. Another advantage of this is if you use a named export you won't need to rename it in your import.